tests/conftest.py

```diff
@@ -16,13 +29,41 @@
 from lbz.resource import Resource
 from lbz.response import Response
 from lbz.router import Router, add_route
-from tests import SAMPLE_PRIVATE_KEY
+from tests.fixtures.rsa_pair import SAMPLE_PRIVATE_KEY, SAMPLE_PUBLIC_KEY
 from tests.utils import encode_token
 
 
 @pytest.fixture(scope="session", name="allowed_audiences")
 def allowed_audiences_fixture() -> List[str]:
-    return os.environ["ALLOWED_AUDIENCES"].split(",")
+    return [str(uuid4()), str(uuid4())]
+
+
+@pytest.fixture(autouse=True, name="patch_environ")
+def patch_environ_fixture(allowed_audiences: List[str]) -> Iterator[None]:
```

Comment:
`environ` is patched, that's true, but I would name this fixture according to what is expected, e.g.:
```python
def setting_initial_lbz_configuration(...) -> Iterator[None]:
```